orc8r/cloud/go/services/bootstrapper/servicers/registration/registration.go

```diff
@@ -0,0 +1,101 @@
+package registration
+
+import (
+	"context"
+	"fmt"
+
+	"magma/orc8r/cloud/go/orc8r"
+	"magma/orc8r/cloud/go/serdes"
+	"magma/orc8r/cloud/go/services/bootstrapper"
+	"magma/orc8r/cloud/go/services/device"
+	models2 "magma/orc8r/cloud/go/services/orchestrator/obsidian/models"
+	"magma/orc8r/cloud/go/services/tenants"
+	"magma/orc8r/lib/go/protos"
+
+	"github.com/go-openapi/strfmt"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type registrationServicer struct {
+}
+
+func NewRegistrationServicer() (protos.RegistrationServer, error) {
+	return &registrationServicer{}, nil
+}
+
+func (rs *registrationServicer) Register(c context.Context, request *protos.RegisterRequest) (*protos.RegisterResponse, error) {
+	nonce := nonceFromToken(request.Token)
+
+	deviceInfo, err := bootstrapper.GetGatewayDeviceInfo(context.Background(), nonce)
+	if err != nil {
+		return formatRegisterResponseError(
+			fmt.Sprintf("could not get device info from nonce %v: %v", nonce, err),
+		), nil
```

Comment:
Returns should always be 1 line -- if takes more than 1 line, create a var on previous lines then return the var